packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -7183,6 +7183,63 @@ void main() {
 
     // On web, using keyboard for selection is handled by the browser.
   }, skip: kIsWeb);
+
+  testWidgets('ignore key event from web platform', (WidgetTester tester) async {
```

Comment:
@justinmc what's the reason that the key event should be ignored on the web platform?

Reply:
Flutter web lets the browser handle text editing, so it doesn't listen to any of these things.

This could be a problem for users that add their own Intents or Shortcuts now that I think about it.  I've added a comment at https://github.com/flutter/flutter/pull/75032#discussion_r577779699 to discuss more.